deploy/kustomize/driver/pmem-csi.yaml

```diff
@@ -155,18 +153,14 @@ spec:
         imagePullPolicy: Always
         image: quay.io/k8scsi/csi-node-driver-registrar:v1.X.Y
         args: [ "-v=3",
-            "--kubelet-registration-path=/var/lib/kubelet/plugins/pmem-csi.intel.com/csi.sock",
+            "--kubelet-registration-path=/var/lib/pmem-csi.intel.com/csi.sock",
             "--csi-address=/csi/csi.sock" ]
         volumeMounts:
-        - name: plugin-socket-dir
+        - name: pmem-state-dir
           mountPath: /csi
```

Comment:
Would it make sense to change the `mountPath` from `/csi` to something else? `/pmem-csi` perhaps?

Reply:
Done